### PR TITLE
EZP-29597: Added component Tabs to UI Guidelines Documentation

### PR DESCRIPTION
**Pull request by inakijv**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | [EZP-29597](https://jira.ez.no/browse/EZP-29597)
| Versions      | 2.2.2

Most important aspect: 
- Added component Tabs to UI Guidelines.

tabs - ez platform developer documentation

### CODE REVIEW

**Comment by inakijv** [comment]
I cannot add reviewers here (yet) pinging @DominikaK & @sunpietro

File: docs/guidelines/components/tabs.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Tabs
+
+We use Tabs component to offer structured views of information to users.
+
+###<div class="mgt-1">Standard tabs</div>
+<div class="mgt-2 ez-guidelines-tabs">
+[[code_example {html}
+<ul class="nav nav-tabs ez-tabs" role="tablist" id="ez-tab-list-location-view">
```
Comment (sunpietro) [inline docs/guidelines/components/tabs.md]: The `id` can be dropped from here.

File: docs/guidelines/components/tabs.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <li class="nav-item">
+        <a class="nav-link active" id="ez-tab-label-location-view" data-toggle="tab" href="#ez-tab-location-view-content" role="tab" aria-controls="ez-tab-location-view-content" aria-expanded="1">View</a>
```
Comment (sunpietro) [inline docs/guidelines/components/tabs.md]: The `id` can be dropped from here as well as `href` attribute. In general, the tab comes from Bootstrap 4. There's nothing eZ-specific in the HTML code structure.
Please update the HTML code by removing not required parts :)

**Comment by inakijv** [comment]
@sunpietro Updated based on review

**Pull request merged**